### DISCUSSION TITLE
Newbie Question: Sending messages via Android app

### DISCUSSION THREAD

**Discussion by swilson317** [meshtastic/Discussions#152] [General]
I just received and configured a WisMesh Pocket as my first Meshtastic device. It seemed that I was receiving messages sent to the default channel. When I attempted to send one, the broken cloud icon appeared next to the message. Do I have something misconfigured?

**Comment by fifieldt** [comment]
Welcome to Meshtastic :) Great work receiving your first messages. It's not uncommon to have only one way communication, especially if the nodes your node is talking to are far away. Try going to an area of higher elevation or closer to another node and see how you go!

**Reply by fifieldt** [reply]
Also, sometimes the messages you send that are marked with a broken cloud do get through, but the confirmation just doesn't make it back.
